static/sass/custom/_linkable-heading.scss

```diff
@@ -0,0 +1,27 @@
+.linkable-heading {
+  position: relative;
+  margin-left: -3.25rem;
+  padding-left: 1rem;
+
+  a {
```

Comment:
I dont like targeting elements in a pattern. Could this be a class or is it required based on the comment above?